packages/react/src/river/RiverAccordion/RiverAccordion.module.css

```diff
@@ -0,0 +1,161 @@
+    grid-template-areas: 'accordion visual';
+  }
+
+  .RiverAccordion__align-end {
+    grid-template-columns: 7fr 4fr; /* 12 columns, minus 1 for the gap */
+    grid-template-areas: 'visual accordion';
+  }
+}
+
+.RiverAccordion__accordionContainer {
+  grid-area: accordion;
+}
+
+.RiverAccordion__visualsContainer {
+  grid-area: visual;
+}
+
+.RiverAccordion__visualsWrapper {
+  overflow: hidden;
+  border-radius: var(--brand-borderRadius-xlarge);
+  display: grid;
+  grid-template-columns: subgrid;
+  grid-template-rows: subgrid;
+}
+
+.RiverAccordion__visualsWrapper > .RiverAccordion__visual {
+  grid-column: 1;
+  grid-row: 1;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  scale: 1.05;
+  transition: opacity var(--brand-animation-duration-default) var(--brand-animation-easing-default),
+    scale var(--brand-animation-duration-default) var(--brand-animation-easing-default);
+}
+
+.RiverAccordion__visualsWrapper > .RiverAccordion__visual > * {
+  width: 100%;
+  object-fit: contain;
+}
+
+/* Show the visual when the corresponding accordion item is open */
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(1)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(1),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(2)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(2),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(3)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(3),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(4)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(4),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(5)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(5),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(6)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(6),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(7)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(7),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(8)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(8),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(9)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(9),
+.RiverAccordion:has(.RiverAccordion__item--open:nth-child(10)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(10),
+/* Just in case all accordions are somehow closed, show the first visual */
+.RiverAccordion:not(:has(.RiverAccordion__item--open)) .RiverAccordion__visualsContainer .RiverAccordion__visual:nth-child(1) {
+  opacity: 1;
+  scale: 1;
+  z-index: 1;
+}
+
+/* Visually hide visuals inside the accordion */
+.RiverAccordion__accordionContainer .RiverAccordion__visual {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  word-wrap: normal;
+  border: 0;
+}
```

Comment:
Could you replace this by using our `visually-hidden` class instead? It's part of the main stylesheet.

Reply:
Not easily unfortunately.

The `.RiverAccordion__visual` element is used in two places; once for the large image that is visible, and again for the visually-hidden image contained within the accordion content. To make this work using `visually-hidden` I'd have to add the `visually-hidden` class to all visuals by default, then selectively remove that class when rendering the large (non-hidden) visual. It's possible but cumbersome, which is why I took this approach.

I can do that if you'd like, but this felt like the simplest approach

Reply:
No worries! Just calling it out in case you weren't aware of that utility class being available. Thanks for clarifying.